docs/manual/introduction.tex

```diff
@@ -751,6 +751,10 @@
   \refqualclass{framework/qual}{AnnotatedFor} annotation, use unchecked code
   default annotations and suppress all type-checking warnings; see
   Section~\ref{compiling-libraries}.
+\item \<-AmergeStubs>
```

Comment:
Would you please move this option above `\<-AuseDefaultsForUncheckedCode=source>`